src/Api.Rest.Dtos.Tests/Data/AttributeDtoTest.cs

```diff
@@ -0,0 +1,300 @@
+﻿namespace Zeiss.PiWeb.Api.Rest.Dtos.Tests.Data
+
+{
+	#region usings
+
+	using System;
+	using FluentAssertions;
+	using NUnit.Framework;
+	using Zeiss.PiWeb.Api.Rest.Dtos.Data;
+
+	#endregion
+
+	[TestFixture]
+	public class AttributeDtoTest
+	{
+		#region methods
+
+		[Test]
+		public void GetDoubleValue_HappyPath_ValueCorrect()
+		{
+			var attribute = new AttributeDto( 1, "1.23" );
+
+			var actualResult = attribute.GetDoubleValue();
+
+			actualResult.Should().BeOfType( typeof( double ) );
+			Assert.AreEqual( actualResult, 1.23 );
```

Comment:
This can be rewritten as `Assert.That( actualResult, Is.EqualTo( 1.23 ))`. The advantage I that the error message for failing tests is much clearer and more useful.

Reply:
The same applies to the other Assert statements.